docker/Dockerfile.build.aarch64-linux

```diff
@@ -44,8 +44,8 @@ RUN LIBZ_VERSION=1.2.11 && \
     rm -rf /tmp/zlib-${LIBZ_VERSION}
 
 # CMake
-RUN CMAKE_VERSION=3.13 && \
-    CMAKE_BUILD=3.13.5 && \
+RUN CMAKE_VERSION=3.18 && \
```

Comment:
Why we need to bump this up?

Reply:
And you need to do that in conda as well.

Reply:
To use CMakeDependentOption

Reply:
Sorry, I see it's not necessary. I'll revert